data/pages/governance.md

```diff
@@ -91,6 +95,25 @@ It is anticipated that the Projects themselves will be self-managing and will
 resolve issues within their communities, without recourse to the Project Leader. Where
 the Project Leader needs to become involved, he/she will act as arbitrator.
 
+#### A.1 Voting process
+
+These rules were approved by the Coq Core Team on ???.
```

Comment:
Did you intend to merge with the missing date?
And is there a point to having this sentence in the middle of the governance document?

Reply:
Not anymore I guess.

Reply:
Fixed as well